src/Controller/HealthCheckController.php

```
@@ -62,9 +62,14 @@ public function getHealth(): JsonResponse
           'timestamp' => date($this->dateFormat),
         ];
 
-        foreach ($this->healthCheckers as $healthService) {
-            $info = $healthService->checkHealth();
-            if (!$info->getStatus()) {
+        foreach ($this->healthCheckers as $healthChecker) {
+            if (!$healthChecker instanceof HealthCheckInterface) {
+                continue;
```

Can you raise an custom invalid parameter?